pulp_2to3_migration/app/migration.py

```diff
@@ -297,8 +302,7 @@ def create_repo_version(migrator, pulp3_repo_name, pulp2_repo, pulp3_remote=None
         pulp2_repo.save()
         return
 
-    repository_class = migrator.pulp3_repository
-    pulp3_repo = repository_class.objects.get(name=pulp3_repo_name)
+    pulp3_repo = pulp2_repo.pulp3_repository
```

Comment:
Is this required for the fix? Or is it a refactor, because we now have a relation with pulp3_repo and this code was written way before that?